src/sqlfluff/core/parser/grammar/anyof.py

```diff
@@ -1,51 +1,71 @@
 """AnyNumberOf, OneOf, OptionallyBracketed & AnySetOf."""
 
-from typing import List, Optional, Tuple
+from typing import FrozenSet, List, Optional
+from typing import Sequence as SequenceType
+from typing import Tuple, Union, cast
```

Comment:
Any reason for multiple calls to the same package?

Reply:
The `isort` library did this automatically 🤔 . It's set up using the `--profile black`, which is described here (https://pycqa.github.io/isort/docs/configuration/profiles.html). I don't hate this layout, but I think it's driven by the `Sequence as SequenceType` line in there. Without that it reverts to the normal format.

This feels like it might be a but in `isort` but given it's being automatically applied. I'm not against it. Views?